integration-tests/build.gradle

```diff
@@ -174,6 +174,41 @@ task runAllIntegrationTests(type: Test) {
     outputs.upToDateWhen { false }
 }
 
+task runAllIntegrationTestsForZoweTestingOnZos(type: Test) {
+    // This task is intended to run on z/OS systems with some limitations:
+    // Only 1 Gateway
+    // z/OSMF Authentication provider only
+    // No support for SAF ID Tokens
+
+    group "Integration tests"
+    description "Run all integration tests for Zowe testing on z/OS (limited)"
+
+    //systemProperty "environment.config", "-sys"
```

Comment:
do we need this line?

Reply:
I'll try to replace this with a property, so the source file can be customized, otherwise I'll just remove it